packages/g/gnome-control-center/files/0001-Fix-binary-location-of-passwd.patch

```diff
@@ -8,13 +8,13 @@ Content-Transfer-Encoding: 8bit
 
 Signed-off-by: Abdulkadir Furkan Şanlı <me@abdulocra.cy>
 ---
- panels/user-accounts/run-passwd.c | 2 +-
+ panels/system/users/run-passwd.c | 2 +-
  1 file changed, 1 insertion(+), 1 deletion(-)
 
-diff --git a/panels/user-accounts/run-passwd.c b/panels/user-accounts/run-passwd.c
```

Comment:
`/usr/bin/passwd` is now the binary location, and `/bin/passwd` a symlink to it. This patch can probably be dropped.